### PR TITLE
librespot: update to f037e46

### PR DESCRIPTION
**Pull request by henri-funck**

### CODE REVIEW

File: packages/addons/service/librespot/changelog.txt
Comment (heitbaum) [inline packages/addons/service/librespot/changelog.txt:1]: No requirement for changelog.txt updates in master (only stable)

Comment (henri-funck) [reply packages/addons/service/librespot/changelog.txt:1]: ok

**Comment by CvH** [comment]
@henri-funck please rename the commit to `librespot: update to f037e46` to keep our in house format

**Comment by CvH** [comment]
@henri-funck I saw you changed the name of the PR, please change the commit message too 
`git commit --amend -m "librespot: update to f037e46"`

**Review by CvH — APPROVED** [review]

**Pull request merged**